wolfssl/internal.h

```diff
@@ -1123,12 +1183,29 @@ enum {
 
 /* set minimum DH key size allowed */
 #ifndef WOLFSSL_MIN_DHKEY_BITS
-    #ifdef WOLFSSL_MAX_STRENGTH
+    #if defined(WOLFSSL_HARDEN_TLS) && !defined(WOLFSSL_HARDEN_TLS_NO_PKEY_CHECK)
```

Comment:
I didn't think we allowed DH cipher suites.
Do we need to set the minimum DH bits?

Reply:
We don't when just using `WOLFSSL_HARDEN_TLS` but if `WOLFSSL_HARDEN_TLS_ALLOW_ALL_CIPHERSUITES` would also be defined then DH ciphersuites could be negotiated.

Reply:
OK